chat-plugins/trivia.js

```diff
@@ -1358,12 +1675,17 @@ module.exports = {
 	commands: {
 		trivia: commands,
 		ta: commands.answer,
-		// TODO: this is ugly as all hell, split it into sections
+		wl: {
```

Comment:
`/wl` should just be an alias for `/trivia` as a whole instead of cherrypicking which commands to alias